content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "How to check if a validator is missing blocks"
+date: 2023-03-11
+author: espanicon
+slug: how-to-check-if-a-validator-is-missing-blocks
+description: Learn how to check if a validator is missing blocks consecutively.
+draft: false
+tags:
+- node-management
+- javascript
+---
+
```

Comment:
Please link to the full tutorial in the docs in the top or bottom of this tutorial